src/ARK/Transaction.cpp

```diff
@@ -0,0 +1,59 @@
+#include "Transaction.h"
+#include "../Data.h"
+#include "../BinaryCoding.h"
+#include "../Base58.h"
+#include "../Hash.h"
+#include "../HexCoding.h"
+#include <nlohmann/json.hpp>
+
+
+using json = nlohmann::json;
+
+using namespace TW;
+
+ARK::Transaction::Transaction(){
+    type=0; // transfer
+    fee = 10000000; // default fee
```

Comment:
did you consider to expose these values to protobuf model? so developers have the ability to specify this

Reply:
Fee is exposed. Type has to be 0 for transfer. I thought we are limited to transfers only.